### DISCUSSION TITLE
Console UI for syncing assets across environments and locally

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cloud#5] [Feature Requests & Enhancements]
Currently, if you want to sync access across environments, you have to use the AWS CLI (`aws sync`) or manually via something like Transmit or Cyberduck.

It would be nice to have an option in the Console UI for this.

It would limit the need for users to deal with S3 credentials as much and potentially solve the major pain points of https://github.com/craftcms/cloud/discussions/4

hat tip to @shennyg

**Comment by timkelty** [comment]
@shennyg @angrybrad can you clarify what you mean by "and locally"?

Syncing between envs makes sense, but to/from local is a bit different.
Are you looking for a UI where you can upload and download your assets directly from Console? I don't think we'd want to reproduce a full-blown file explorer in the browser, but maybe a single place to upload/download a zip of all the assets?

**Reply by shennyg** [reply]
I'm good with only across ENVs.

**Reply by timkelty** [reply]
Ultimately I think up and down from local is probably going to be best via CLI, eg `ddev craft cloud/assets/sync --from production`.
